samples/subsys/fs/littlefs/sample.yaml

```diff
@@ -38,5 +42,35 @@ tests:
     build_only: true
     platform_allow: stm32h747i_disco_m7
     extra_args:
-      - OVERLAY_CONFIG=boards/stm32h747i_disco_m7.conf
+      - OVERLAY_CONFIG=boards/stm32h747i_disco_m7_blk.conf
+      - CONF_FILE=prj_blk.conf
+  sample.filesystem.littlefs.nucleo_h7a3zi_q_flash:
+    build_only: true
+    platform_allow: nucleo_h7a3zi_q
+    extra_args:
+      - OVERLAY_CONFIG=boards/nucleo_h7a3zi_q.conf
+      - CONF_FILE=prj.conf
```

Comment:
These files will be automatically used if present. If these were the only reasons to add this test config, it could be removed and `nucleo_h7a3zi_q` could be added in `sample.filesystem.littlefs`